engine/src/main/java/org/terasology/world/block/entity/BlockCommands.java

```diff
@@ -307,36 +299,19 @@ public String giveBlock(
                     return giveBlock(blockManager.getBlockFamily(new BlockUri(def.get().getUrn())), quantity, sender);
                 }
             } else {
-                return "No block found for '" + uri + "'";
+                return "Could not find an item or block matching \"" + uri + "\"";
```

Comment:
Showing the message "could not find item ..." within a method called `giveBlock` seems unintuitive. However, I'm not 100% sure how this can be resolved better. 
Since `giveBlock` is not a command anymore, you might use it to return the block URN if found, and do the error handling in the `give` or `giveBulkBlock` commands. Which, in turn, would mean duplication of the code... *sigh*